src/util.cc

```diff
@@ -318,7 +321,7 @@ int ReadFile(const string& path, string* contents, string* err) {
   // This makes a ninja run on a set of 1500 manifest files about 4% faster
   // than using the generic fopen code below.
   err->clear();
-  HANDLE f = ::CreateFileA(path.c_str(), GENERIC_READ, FILE_SHARE_READ, NULL,
+  HANDLE f = ::CreateFileW(Utf8ToWide(path).c_str(), GENERIC_READ, FILE_SHARE_READ, NULL,
```

Comment:
@jlonnberg 
Change the function we're in to take a StringPiece instead of a std::string const&

Any function that you modify, if, after your change, it only uses the incoming string after it's been converted to WString, change the input argument to the function from std::string const& to StringPiece.